tools/maxquant/macros.xml

```diff
@@ -882,6 +882,7 @@
                 }
             </citation>
             <citation type="doi">10.1021/acs.jproteome.5b00780</citation>
```

Comment:
should the old one be dropped?

Reply:
@foellmelanie what do you think? keep both?

Reply:
@foellmelanie has agreed to leave both references in!